README.md

```diff
@@ -9,8 +9,8 @@ It is designed to build as a Debian package.
 ## Building under stretch
 
 ```bash
-$ sudo apt-get install build-essential debhelper librtlsdr-dev pkg-config dh-systemd libncurses5-dev libbladerf-dev
-$ dpkg-buildpackage -b
+sudo apt-get install build-essential debhelper librtlsdr-dev pkg-config dh-systemd libncurses5-dev libbladerf-dev
```

Comment:
Not a fan of this

Reply:
Removing the prompt indicator? The reasoning here is that it's easy to quickly select the entire block and paste it to run these commands. I tried searching for Markdown that allows the appearance of the `$` without interfering with it.

Reply:
I don't think encouraging blind copy&paste of blocks of commands is a good idea; in the past people have done this without really looking at what they're doing and then get confused by errors from the last command because an earlier commands actually failed.

Reply:
Fair enough! Looks like the partial merge will suffice.